queries.sql

```diff
@@ -1,3 +1,76 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon%';
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+
+SELECT * FROM animals WHERE neutered IS true AND escape_attempts < 3;
+
+SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
+
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+
+SELECT * FROM animals WHERE neutered IS true;
+
+SELECT * FROM animals WHERE name NOT LIKE 'Gabumon';
+
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+-- New Queries T1
+BEGIN; 
+UPDATE animals SET  species = 'unspecified'; 
+SELECT species FROM animals;
+ROLLBACK;
+
+SELECT species FROM animals;
+
+-- T2
+BEGIN; 
+UPDATE animals SET  species = 'digimon' 
+WHERE name LIKE '%mon%'; 
+
+UPDATE animals SET  species = 'pokemon' 
+WHERE species is null ;
+SELECT species FROM animals; 
+COMMIT; 
```

Comment:
Great job @AhmadiGu 🤝

- [ ] You may as well add the screenshot of this transaction to the Pull Request as specified in the project requirements.
![thirdt](https://user-images.githubusercontent.com/63917725/215763307-2ff577fc-c43f-4c22-97ee-b458197b2458.PNG)